### PR TITLE
Create config when launching interactive console without config

### PR DESCRIPTION
**Pull request by konjac**
If there is no run config selected when launching interactive console, a new runconfig will be created determined by the default spark type(If there is no default spark type, there will be popup menu)

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/action/RunTwoSequentialActions.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ */
+
+package com.microsoft.azure.hdinsight.spark.run.action
+
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.ex.ActionManagerEx
+import com.microsoft.azure.hdinsight.common.logger.ILogger
+import com.microsoft.azuretools.ijidea.utility.AzureAnAction
+
+open class RunTwoSequentialActions(private val firstActionId: String, private val secondActionId: String): AzureAnAction(), ILogger {
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/action/RunTwoSequentialActions.kt]: We can simple the class name, and add more action items later.

```suggestion
open class SeqActions(private val firstActionId: String, private val secondActionId: String): AzureAnAction(), ILogger {
```

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/action/RunTwoSequentialActions.kt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    override fun onActionPerformed(anActionEvent: AnActionEvent?) {
+        if(anActionEvent != null) {
+            val firstAction = ActionManagerEx.getInstance().getAction(firstActionId)
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/action/RunTwoSequentialActions.kt]: Build an action list and for each perform?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/console/RunSparkScalaConsoleAction.kt [hunk 3]
```diff
@@ -61,7 +66,13 @@ abstract class RunSparkScalaConsoleAction
             return
         }
 
-        val batchConfigurationType = LivySparkBatchJobRunConfigurationType.getInstance()
+        val batchConfigurationType = SelectSparkApplicationTypeAction.getRunConfigurationType()
+        if(batchConfigurationType == null){
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/console/RunSparkScalaConsoleAction.kt]: Needs some spaces.

```suggestion
        if (batchConfigurationType == null) {
```

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/action/RunTwoSequentialActions.kt (hunk 2 above)
Comment (konjac) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/action/RunTwoSequentialActions.kt]: OK. New commit will switch args to list.

**Pull request merged**